pkg/cluster/reconcile.go

```diff
@@ -114,7 +115,7 @@ func (c *Cluster) reconcileMembers(running etcdutil.MemberSet) error {
 	}
 
 	// Reinitialize the cluster again
-	if err := c.setup(); err != nil {
+	if err := c.create(); err != nil {
```

Comment:
Calling setup would get cluster into failed state. Create initializes it instead, bootstrapping seed member and setting appropriate status.